lib/gpu/lib/src/render_pass.dart

```diff
@@ -326,6 +338,11 @@ base class RenderPass extends NativeFieldWrapperClass1 {
         targetFace.index);
   }
 
+  void setScissor(Scissor scissor) {
+    scissor._validate();
```

Comment:
Please call the validation within an assert (as is done for `RenderTarget` validation in `RenderPass._`). Apologies for the delay, but I think this should be the last one. :)